src/DPORDriver.cpp

```diff
@@ -325,10 +327,13 @@ DPORDriver::Result DPORDriver::run_rfsc_sequential() {
     if(conf.print_progress_estimate && (computation_count+1) % 100 == 0){
       estimate = std::round(TB.estimate_trace_count());
     }
-    if((computation_count+1) % 1000 == 0){
-      /* llvm::ExecutionEngine leaks global variables until the Module is
-       * destructed */
-      mod = parse();
+    if((computation_count+1) % 1024 == 0){
```

Comment:
Wouldn't it be better to factor out the various occurrences of `1024` in this file into a symbolic constant (via a `#define` or a `const`)?  Perhaps even the whole check `*... % 1024 == 0)` can be factored out (and be changed to use some bit operation instead of modulo, which is quite expensive).